### DISCUSSION TITLE
HomeAssistant - automation on errors?

### DISCUSSION THREAD

**Discussion by owenashurst** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#568] [Q&A]
I've noticed today that my filter needed replacing since it threw an error code on the pump (E02). In home assistant however, I can't seem to find a trigger on "errors". Is it somewhere else? I'd like to notify my phone on any errors so I can sort them there and then.

Thanks!

**Comment by visualapproach** [comment]
Somewhere in this forum there is an example of how to do that. The display characters are sent as char1, 2 and 3. Basically you need to make sensors for char1 through char3 in the mqtt message topic, and trigger on ascii 'e' on char1.
Did that make any sense?

**Comment by chunkysteveo** [comment]
I thought we also added the mqtt 'has error' toppic? Or did that disappear?

I originally set up the error notification based on the 4 wire character value changing to 'E', and it sent me a mobile notification.

**Reply by Techtobi83** [reply]
in my FHEM integration of this project, i can see an reading (=variable, don´t know the correct name of this in HA) called "ERR" with actual value "0". So i would guess this is an indicator for errors?

**Reply by chunkysteveo** [reply]
Yeah, i'm sure that changes to "1" when an error occurs. It was added to replace the use case having to read the character display values for an "E" value.

**Reply by Techtobi83** [reply]
simply integrated "ERR" status in my smarthome server, works great. when an error occurs, I get a message on my wrist. 
![072cc183-eebe-43da-a49a-59f187efbc3c](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/86073272/19a37f2e-6cdf-4816-8dd7-d827b0bf275d)
